ladybug/designday.py

```diff
@@ -0,0 +1,1109 @@
+from .location import Location
+from .futil import write_to_file
+from .dt import DateTime
+from .header import Header
+from .datacollection import DataCollection
+from .datatype import DataPoint
+from .analysisperiod import AnalysisPeriod
+from .sunpath import Sunpath
+
+from .skymodel import ashrae_revised_clear_sky
+from .skymodel import ashrae_clear_sky
+from .skymodel import energyplus_horizontal_infrared
+
+from .psychrometrics import rel_humid_from_db_dpt
+from .psychrometrics import dew_point_from_db_hr
+from .psychrometrics import dew_point_from_db_enth
+from .psychrometrics import dew_point_from_db_wb
+
+import math
+import os
+import re
+import codecs
+import platform
+
+day_types = ['SummerDesignDay', 'WinterDesignDay', 'Sunday', 'Monday',
```

Comment:
Should we move all this inside the Class as a class CONSTANT? Or you can put them under a property.
```Python
@property
def day_types:
    return ('SummerDesignDay', 'WinterDesignDay', 'Sunday', 'Monday', ...)
```
```

Reply:
I've now integrated all of these constants on the objects where they are used.  You are right that this is a lot cleaner.